src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -31,6 +32,7 @@ See the LICENSE file in the project root for more information.
   </ItemGroup>
 
   <ItemGroup>
+    <NativeLibrary Condition="'$(IlcMultiModule)' == 'true' and $(NativeCodeGen) == ''" Include="$(FrameworkLibPath)\libframework.a" />
```

Comment:
Hm I can probably just move it into Microsoft.NETCode.Native.targets before it imports the OS-specific props files.